packages/firestore/src/model/path.ts

```diff
@@ -366,3 +366,57 @@ export class FieldPath extends BasePath<FieldPath> {
     return new FieldPath([]);
   }
 }
+
+export class DocumentKey {
```

Comment:
DocumentKey doesn't seem to have any circular references with paths--notably this didn't result in any import changes in `path.ts`. This class is separate on other platforms and it seems like it should be separate here as well. What's the reason to move this?

Reply:
I swear it did when I looked at it, but it seems to have magically disappeared. Reverted this change.